doc/cprover-manual/contracts-decreases.md

```diff
@@ -1,3 +1,179 @@
-# Decreases Clause
+# Decreases Clauses
 
-TODO: Document `__CPROVER_decreases`
+A _decreases_ clause specifies a measure that must strictly decrease at every iteration of a loop.
```

Comment:
Do you have a link to any external resources with more details in the theory behind decreases clauses? It could be useful to add it here.

Reply:
Good point; I'll add some resources.